File: ProjectManager.Domain/Entities/ProjetoResponsavel.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace ProjectManager.Domain.Entities
{
    [Table("ProjetoResponsavel")]
    public class ProjetoResponsavel : _EntityBase
    {
        [Key]
        public decimal Id { get; set; }
        public decimal ProjetoId { get; set; }
        public decimal? ResponsavelId { get; set; }
        public Int16 AtivoId { get; set; }
        public Int16 ExcluidoId { get; set; }
        public Guid IdExterno { get; set; }

        public virtual SimNao Excluido { get; set; }
        public virtual SimNao Ativo { get; set; }
        public virtual Responsavel Responsavel { get; set; }
        [JsonIgnore]
        public virtual Projeto Projeto { get; set; }
    }
}

File: ProjectManager.Web/src/app/shared/interfaces/projeto/projeto-responsavel.ts
import { Entidade } from '@interfaces/geral/entidade';
import { Responsavel } from '@interfaces/geral/responsavel';
import { Projeto } from './projeto';

export interface ProjetoResponsavel {
    id: number;
    projetoId: number;
    responsavelId?: number;
    ativoId: number;
    excluidoId: number;
    idExterno: string;
    excluido?: Entidade;
    ativo?: Entidade;
    projeto?: Projeto;
    responsavel?: Responsavel;
}
